### DISCUSSION TITLE
找回文件

### DISCUSSION THREAD

**Discussion by RihuiAn** [electerm/electerm#2716] [Q&A]
在electerm中误删除了文件的一个文件夹，回收站和恢复软件都找不回，请问有什么好的办法找回吗？

**Comment by zxdong262** [comment]
本质也是调用系统删除命令，没有其他的特别处理，跟命令行`rm -rf xxx`效果一致。

**Reply by RihuiAn** [reply]
谢谢您的回复，但是命令行rm -rf xxx删除的文件我可以通过inode编号找回，为什么在electerm中误删除的文件无法找回呢？

**Reply by zxdong262** [reply]
> 谢谢您的回复，但是命令行rm -rf xxx删除的文件我可以通过inode编号找回，为什么在electerm中误删除的文件无法找回呢？

详细情况是怎么样的？软件版本，操作系统，详细的操作步骤。
